dataikuapi/dss/admin.py

```diff
@@ -365,6 +378,79 @@ def save(self):
         self.client._perform_empty("PUT", "/current-user", body = self.settings)
 
 
+class DSSUserActivity(object):
```

Comment:
You probably need to modify the doc, the same way I did here:
https://github.com/dataiku/dss-doc/pull/603/files